From 13191f413a4dbff98094fe407cb592ae9723f956 Mon Sep 17 00:00:00 2001
From: Lennart Poettering <lennart@poettering.net>
Date: Mon, 7 Jul 2014 18:45:07 +0200
Subject: [PATCH] man: drop references to the --priviliged command line option
 which has been removed a while back

(cherry picked from commit ed3e4a3fd96891b5e7015723978e78cd21efd4fe)
---
 man/hostnamectl.xml | 8 --------
 man/localectl.xml   | 8 --------
 2 files changed, 16 deletions(-)

diff --git a/man/hostnamectl.xml b/man/hostnamectl.xml
index 71973fde91..4e456eb8f9 100644
--- a/man/hostnamectl.xml
+++ b/man/hostnamectl.xml
@@ -102,14 +102,6 @@
                         </varlistentry>
 
                         <varlistentry>
-                                <term><option>-P</option></term>
-                                <term><option>--privileged</option></term>
-
-                                <listitem><para>Acquire privileges via PolicyKit
-                                before executing the operation.</para></listitem>
-                        </varlistentry>
-
-                        <varlistentry>
                                 <term><option>--static</option></term>
                                 <term><option>--transient</option></term>
                                 <term><option>--pretty</option></term>
diff --git a/man/localectl.xml b/man/localectl.xml
index c2e79a2d71..9c32c794aa 100644
--- a/man/localectl.xml
+++ b/man/localectl.xml
@@ -90,14 +90,6 @@
                         </varlistentry>
 
                         <varlistentry>
-                                <term><option>-P</option></term>
-                                <term><option>--privileged</option></term>
-
-                                <listitem><para>Acquire privileges via PolicyKit
-                                before executing the operation.</para></listitem>
-                        </varlistentry>
-
-                        <varlistentry>
                                 <term><option>--no-convert</option></term>
 
                                 <listitem><para>If
